.github/workflows/build-artifacts.yml

```
@@ -16,21 +18,24 @@ jobs:
         uses: actions/setup-dotnet@v1
         with:
           dotnet-version: '3.1.x'
+          source-url: https://nuget.pkg.github.com/graphql-dotnet/index.json
+        env:
+          NUGET_AUTH_TOKEN: ${{secrets.GITHUB_TOKEN}}
       - name: Install dependencies
         working-directory: src
         run: dotnet restore
       - name: Build solution (release config)
         working-directory: src
-        run: dotnet build --no-restore -c Release -p:NoWarn=CS1591 -p:VersionSuffix=${{ github.run_id }}
+        run: dotnet build --no-restore -c Release -p:NoWarn=CS1591 -p:VersionSuffix=$GITHUB_RUN_NUMBER
       - name: Pack solution (release config)
         working-directory: src
-        run: dotnet pack --no-restore -c Release -p:VersionSuffix=${{ github.run_id }} -o out
+        run: dotnet pack --no-restore -c Release -p:VersionSuffix=$GITHUB_RUN_NUMBER -o out
       - name: Publish artifacts
         uses: actions/upload-artifact@v2
         with:
           name: Nuget packages
           path: |
             src/out/*.nupkg
```

```suggestion
            src/out/*
```
In case we publish symbols separately (currently embedded, I think)